requirements/prod.txt

```diff
@@ -5,3 +5,4 @@ flask-wtf==0.8
 flask-sqlalchemy==0.16
 sqlalchemy-migrate==0.7.2
 cef==0.5
+simplejson
```

Comment:
This belongs in compiled.txt, because our webheads have the C version installed on them. prod.txt is for things that go in the embedded vendor library.